examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -0,0 +1,634 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-01-02T01:57:45.309973Z",
+     "start_time": "2021-01-02T01:57:45.285024Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[0, 1, 2, 3]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ipyparallel as ipp\n",
+    "rc = ipp.Client(profile=\"bluefog\")\n",
+    "rc.ids"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you saw the `ids` is an empty list, this means ipyparallel cannot connect it with the clusters. You can stop the `ibfrun` by `ctrl+c`. Kill all remainning `ipcontroller` or `mpi` processes through `ps` if necessary. Then, try to rerun the `ibfrun` again."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Parallel execution through `px`\n",
+    "\n",
+    "`px` -- parallel execution is a [magic command](https://ipython.readthedocs.io/en/stable/interactive/magics.html) in IPython environment similar as the `%matplotlib` or `%%timeit`. If you are not familar with magic command, you can think it as a special command in IPython that will change the behavior of IPython to interprete the following python code. As the same for all magic commands, `%px` is for one line and `%%px` is for whole cell. For example \n",
+    "`%px print(\"Hello world\")` will put the `print(\"Hello world\")` to run under the `px` environment. And, if you put the `%%px` at the top of one cell, the rest code in that cell all will be run under the `px` environment.\n",
+    "\n",
+    "The code under the `px` env will be executed into all each worker in parallel instead of running in this notebook.\n",
+    "It is important to rememebr that each worker are one seperated IPython environment as well. The following is hello-world example to work with bluefog in jupyter notebook, which is already sufficient for most  usage cases. You can find more advanced usage of `px` in the [document page](https://ipyparallel.readthedocs.io/en/latest/magics.html).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-01-02T01:57:46.578543Z",
+     "start_time": "2021-01-02T01:57:46.536076Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[stdout:0] Hello, I am 2 among 4 processes\n",
+      "[stdout:1] Hello, I am 3 among 4 processes\n",
+      "[stdout:2] Hello, I am 0 among 4 processes\n",
+      "[stdout:3] Hello, I am 1 among 4 processes\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%px\n",
+    "import bluefog.torch as bf\n",
+    "\n",
+    "bf.init()\n",
```

Comment:
Consider emphasizing the usage of bf.init().

Reply:
added